python/cuml/cluster/kmeans.pyx

```diff
@@ -205,22 +205,25 @@ class KMeans(Base):
     random_state : int (default = 1)
         If you want results to be the same when you restart Python, select a
         state.
-    init : {'scalable-kmeans++', 'k-means||' , 'random' or an ndarray}
-           (default = 'scalable-k-means++')
+    init : 'scalable-kmeans++', 'k-means||' , 'random' or an ndarray (default = 'scalable-k-means++')  # noqa
```

Comment:
not sure, but is this suppose to be here `#noqa`?

Reply:
no, spaces are required for the `# noqa` to be read by flake8